### PR TITLE
Cyware Submission - CAR-2021-01-001

### PR DESCRIPTION
**Pull request by ikiril01**
Submitted by Cyware Labs. Thanks to @kp625544 and team!

### CODE REVIEW

File: analytics/CAR-2021-01-001.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Identifying Internal hosts and services for lateral movement
```
Comment (pcmcpherson) [inline analytics/CAR-2021-01-001.yaml]: Title is too long and vague; change to "Port scanning activity"

File: analytics/CAR-2021-01-001.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    coverage: Moderate
+    subtechniques:
+      - No sub-techniques
+    tactics:
+      - TA0007
+implementations:
+- name: Splunk search - Identifying Internal hosts and services for lateral movement
+  description: It should be noted that when a host/ port/ service scan is performed from a compromised machine, a single machine makes multiple calls to other hosts in the network to identify live hosts and services. This can be detected using the following query
+  code: |-
+    sourcetype='firewall_logs' dest_ip = 'internal_subnet' | stats dc(dest_port) as pcount by source_ip | where pcount >5
```
Comment (pcmcpherson) [inline analytics/CAR-2021-01-001.yaml]: source_ip should be src_ip, the standard fieldname in Splunk

File: analytics/CAR-2021-01-001.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  data_model: Sysmon native
+  type: Splunk
+data_model_references:
+  - flow/start/dest_ip
+  - flow/stop/dest_ip
```
Comment (pcmcpherson) [inline analytics/CAR-2021-01-001.yaml]: remove this line, as CAR does not have this field, and we're not using it in this analytic.

**Review by pcmcpherson — CHANGES_REQUESTED** [review]
Hi, thanks for the PR. Sorry for takin so long to get back to you; we've been understaffed in this department recently. 
Just a couple general comments: 
1. Make sure you validate your YAML syntax using a yaml linter/validator. 
2. You don't need to submit the .md file for each analytic; just the yaml will do. Our process will convert the yaml into .md for the website.

**Comment by kp625544** [comment]
Changes submitted in PR - https://github.com/mitre-attack/car/pull/90

**Review by pcmcpherson — APPROVED** [review]

**Pull request merged**